### DISCUSSION TITLE
What is the best way to handle components that is not used anymore (E.g: Model, Controller)?

### DISCUSSION THREAD

**Discussion by sidoqi** [loco-rs/loco#274] [Q&A]
Any best practices?

**Comment by jondot** [comment]
You mean stuff you don't use at all?
You can safely delete those from the project

**Reply by sidoqi** [reply]
Noted, so we need to delete them manually. Thanks for the answer

**Reply by sidoqi** [reply]
But especially for entities that might have relationship with another one, we need to recompile again for the orm after the deletion, right?

How can we make the process seamless without any manual works? So the db migration can be handled automatically
